### ISSUE TITLE
Problem with oldbookdown?

### ISSUE THREAD

**Issue by MansMeg** [hadley/ggplot2-book#118]
Hi!

I get the following error message when I try to compile the PDF in R-Studio?

```
Rscript book/render-tex.R data-manip.rmd
Error in loadNamespace(name) : there is no package called ‘oldbookdown’
Calls: :: ... tryCatch -> tryCatchList -> tryCatchOne -> <Anonymous>
Execution halted
make: *** [book/tex/data-manip.tex] Error 1

Exited with status 2.
```

And `oldbookdown`` is depricated?

**Comment by morningfin** [comment]
I also have this problem. How should we fix it?

**Comment by MansMeg** [comment]
I installed it allthough it was depricated. It works as a hack.

**Comment by brophyj** [comment]
I have the same issue. Being totally unsophisticated how do i install this package, it doesn't seem to exist on CRAN.

**Comment by MansMeg** [comment]
It should work to run:
devtools::install_github("hadley/oldbookdown")

**Comment by brophyj** [comment]
Thank you, library installation worked fine.
Unfortunately still can't completely execute the "make" command. Will need to try and track down the error (see below)

Latexmk: This is Latexmk, John Collins, 10 Nov 2013, version: 4.39.
**** Report bugs etc to John Collins <collins at phys.psu.edu>. ****
Latexmk: applying rule 'pdflatex'...
Rule 'pdflatex': Rules & subrules not known to be previously run:
   pdflatex
Rule 'pdflatex': The following rules & subrules became out-of-date:
      'pdflatex'
------------
Run number 1 of rule 'pdflatex'
------------
------------
Running 'pdflatex  -interaction=batchmode -recorder  "ggplot2-book.tex"'
------------
sh: pdflatex: command not found
Latexmk: Errors, so I did not complete making targets
Collected error summary (may duplicate other messages):
  pdflatex: (Pdf)LaTeX failed to generate the expected log file 'ggplot2-book.log'
Latexmk: Did not finish processing file 'ggplot2-book.tex':
   (Pdf)LaTeX failed to generate the expected log file 'ggplot2-book.log'
Latexmk: Use the -f option to force complete processing,
 unless error was exceeding maximum runs of latex/pdflatex.
make: *** [book/ggplot2-book.pdf] Error 12

**Comment by hadley** [comment]
Should be fixed by #142
